build.gradle.kts

```diff
@@ -75,17 +75,20 @@ buildscript {
 plugins {
     `jvm-module`
     protobuf
+    ksp
+    `module-testing`
 }
 
 // Cannot use `id()` syntax for McJava because it's not yet published to the Plugin Portal
-// and is added to the build classpath via `buildScript` block above.
+// and is added to the build classpath via the `buildScript` block above.
 apply(plugin = "io.spine.mc-java")
 
 apply<IncrementGuard>()
 apply<VersionWriter>()
 
+apply<BomsPlugin>()
```

Comment:
Maybe we can go without `Plugin` suffix? Other local plugins don't use it.

It can be `BonEnforcer` or `BomAlignment`.

Reply:
This is an experiment in progress. It's not enforcing BOM already. Let's see how it goes and then select the name. We've got a lot of code around the dependencies.

There's `Boms` object, BTW. So, the plugin has to have the suffix.